From 8517a17a2161f8cd6b9f2fdf1d25d6e6193ed7bf Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 31 Jul 2018 07:11:55 +0300
Subject: [PATCH] dali [uboot]: update uboot load address at its file

SPL use uboot load address ad temp for the itb file and it clashes
some test loadable data
---

diff --git a/board/habanalabs/dali/dali_fit_spl.its b/board/habanalabs/dali/dali_fit_spl.its
index 6eae68b..9ce365d 100644
--- a/board/habanalabs/dali/dali_fit_spl.its
+++ b/board/habanalabs/dali/dali_fit_spl.its
@@ -18,7 +18,7 @@
 			type = "standalone";
 			arch = "arm64";
 			compression = "none";
-			load = <0x7f 0xf0010000>;
+			load = <0x7f 0xf0100000>;
 		};
 
 		fdt {
@@ -32,28 +32,28 @@
 			description = "Dali data 0";
 			data = /incbin/("uboot-data0.bin");
 			compression = "none";
-			load = <0x7f 0xf0100000>;
+			load = <0x7f 0xf0000000>;
 		};
 
 		data1 {
 			description = "Dali data 1";
 			data = /incbin/("uboot-data1.bin");
 			compression = "none";
-			load = <0x7f 0xf0110000>;
+			load = <0x7f 0xf0010000>;
 		};
 
 		data2 {
 			description = "Dali data 2";
 			data = /incbin/("uboot-data2.bin");
 			compression = "none";
-			load = <0x7f 0xf0120000>;
+			load = <0x7f 0xf0020000>;
 		};
 
 		data3 {
 			description = "Dali data 3";
 			data = /incbin/("uboot-data3.bin");
 			compression = "none";
-			load = <0x7f 0xf0130000>;
+			load = <0x7f 0xf0030000>;
 		};
 */
 	};
